src/dmd/frontend.h

```diff
@@ -1769,60 +1770,6 @@ struct File final
     }
 };
 
-struct OutBuffer final
```

Comment:
Hmm, although we shouldn't (gcc has obstack for writing data to a buffer) there's still a couple places where we need the OutBuffer definition, ldc too I guess for the same reasons.

Reply:
`dtoh` omits `OutBuffer` because it's declared as a D struct and only used by pointer/ref.
But that's easily fixed by marking it as `extern(C++)`.